vr-wordle/src/Components/ResultsScreen.js

```diff
@@ -1,17 +1,33 @@
-import ResultsGraph from './ResultsGraph';
-import './results.css';
+import ResultsGraph from "./ResultsGraph";
+import { intervalToDuration } from "date-fns";
+import "./results.css";
 
 export default function ResultsScreen(props) {
   const { answer, userScore, gameTime } = props;
+  let userTime = intervalToDuration({ start: 0, end: gameTime });
+  let formattedTime = ``;
+  if (userTime.minutes) {
+    formattedTime +=
+      userTime.minutes === 1 ? `1 minute` : `${userTime.minutes} minutes`;
+  }
+  if (userTime.seconds && userTime.minutes) {
+    formattedTime +=
+      userTime.seconds === 1
+        ? " and 1 second"
+        : ` and ${userTime.seconds} seconds`;
+  } else if (userTime.seconds) {
+    formattedTime +=
+      userTime.seconds === 1 ? "1 second" : `${userTime.seconds} seconds`;
+  }
```

Comment:
great conditional string, could we pull this out to its own function?

Reply:
yeah sure